### DISCUSSION TITLE
6 小時生成致命毒劑：千萬不能遭濫用的 AI

### DISCUSSION THREAD

**Discussion by timchen1216** [1101-GS3073-C/Python-Community-GS3073-C#352] [科技新知]
少人都覺得製毒並不容易，至少需有化學知識。但如果請人工智慧幫忙呢？且 AI 系統「做」出的不是毒品而是毒藥。

以 Fabio Urbina 為第一作者於《Nature Machine Intelligence》發表的論文提到，他任職的公司 Collaborations Pharmaceuticals 最近公布預測毒性的機器學習模型。受邀參加瑞士 NBC（核子、生物和化學）保護研究所會議時，討論尖端化學和生物技術可能產生影響的工具發展，Collaborations Pharmaceuticals 受邀談論人工智慧技術可能濫用的問題。

Urbina 表示，之前似乎沒想過這問題，因幾十年來機器學習模型發現可製藥的新分子，使用電腦和人工智慧是改善人類健康，不是破壞。Collaborations Pharmaceuticals 卻決定探索如何使用 AI 設計有毒分子，之前設計名為 MegaSyn 的商業 de novo 分子生成模型，藉助機器學習模型預測生物活性，尋找人類疾病標靶新治療抑製劑。

這種生成模型通常會懲罰預測毒性並獎勵預測目標活動，之後調整指導模型同時獎勵毒性和生物活性，並使用公共資料庫分子訓練人工智慧。調整後底層生成軟體建立在易獲得的開源軟體上，為了縮小分子範圍，將生成模型推向諸如神經毒劑 VX 類化合物。

VX 是神經毒劑的人造化學戰劑（化學戰劑：戰爭目的、有劇烈毒性、大規模毒害或殺傷敵方人畜和植物等化學物質），毒性強且作用迅速，6~10 毫克 VX 顆粒便足以致命。伺服器啟動後 6 小時內，新模型就生成 4 萬個分子。人工智慧不僅設計出 VX，還設計許多已知化學戰劑，更設計出許多看起來同樣合理的新分子，據預測值看，新分子比已知化學戰劑毒性更強。
![image](https://user-images.githubusercontent.com/95116299/172053983-acd3b75c-0aff-4760-ada9-da983a53f87d.png)
訓練 AI 的資料庫並不包括這些神經毒劑，但反轉機器學習模型，卻將無害生成模型從有用的醫學工具轉為致命的分子產生器。為了避免毒性創建的模型變成「雙面刃」。研究人員越能預測毒性，引導生成模型在主要由致命分子組成的化學空間設計新分子的效果也越強。

Collaborations Pharmaceuticals 沒有評估模型生成的虛擬分子可合成性，也沒有探索如何製造，但過程都有現成商業模式和開源軟體。雖然他們也沒有物理合成任何分子，但全球有數百家商業公司能合成。

這證明一件事：非人類也能製造致命化學武器。

雖然目前要生成可能造成重大傷害的有毒物質或生物製劑仍需要化學或毒理學知識，但加入機器學習模型後，技術門檻大大降低，可能只要編碼和理解模型輸出的能力。商業工具、開源軟體工具和公共資料庫許多資料，都可在沒有監督下使用，利用人工智慧生成有害分子的模型似乎成了「潘朵拉的盒子」。這些分子可輕鬆刪除，但如何創造卻不會消失。

很明顯，必須想辦法避免人工智慧濫用。Urbina 認為，人工智慧化學戰並不會很快出現，但有這種可能。MegaSyn 是商業產品，可控制誰能接觸，未來也可能增加限制，像 OpenAI「GPT-3」語言模型，雖然能隨時使用，但也能切斷使用者存取權。

Urbina 還提到，大學也要加倍努力培訓理科生的道德，範圍可擴大到其他學科，尤其計算機專業學生，讓他們意識人工智慧濫用的可能性。這次嘗試似乎再次印證一句話：技術無罪，造福還是作惡，取決於使用者的目的。

資料來源:https://technews.tw/2022/04/03/collaborations-pharmaceuticals/

工具沒有好壞，人才有

**Comment by lx3211** [comment]
如果用在戰爭上，造成的傷亡一定會比現在更多..

**Comment by 32844583** [comment]
如果有蠻多國家的政府秘密組織有在研發這個，人類的滅亡真的一兩天就可以實現 zzz

**Comment by NcuJerryKao** [comment]
心懷不軌 也可以運用AI 造成更大危害
不知道犯罪 能不能透過AI幫忙 計算出最最最萬無一失的策略

不過 警方也可以 以犯人角度 想如何運用AI犯罪 以此抓捕犯人

 然而 犯人也是可以 預測 警方會預測他

但是 警察已可以 預測 犯人預測他的預測

**Comment by BingRong1217** [comment]
水能載舟 亦能覆舟

**Comment by limitx0** [comment]
AI常常用來預測合成新分子
對化學研究很有幫助
但是有心人士拿去合成毒藥不知道要怎麼避免

**Comment by larry40305** [comment]
如果用在戰爭上，絕對會造成無法控制的後果
